src/providers/consensus/client.py

```diff
@@ -108,7 +110,9 @@ def get_block_details(self, state_id: SlotNumber | BlockRoot) -> BlockDetailsRes
         return BlockDetailsResponse.from_response(**data)
 
     @lru_cache(maxsize=256)
-    def get_block_attestations(self, state_id: SlotNumber | BlockRoot) -> list[BlockAttestation]:
+    def get_block_attestations(
+        self, state_id: SlotNumber | BlockRoot
+    ) -> list[BlockAttestationPhase0 | BlockAttestationElectra]:
```

Comment:
Actually it always returns BlockAttestation

Reply:
BlockAttestationResponse now